pipeline/nodes/basic.py

```diff
@@ -166,73 +175,6 @@ def __call__(self, *data):
 		else:
 			return data
 
-def deep_copy(data):
-	new = {}
-	# Not actually very deep, just shallow copies everything except
-	# the object generated in another thread
-	for (k,v) in data.items():
-		if k != "_LOD_OBJECT":
-			new[k] = v
-	return new
-
-def get_actor_type(ulan, uuid_cache, default="Actor"):
-	if not ulan:
-		return "Actor"
-	s = 'SELECT type FROM actor_type WHERE ulan = :ulan'
-	res = uuid_cache.execute(s, ulan=ulan)
-	v = res.fetchone()
-	if v:
-		return v[0]
-	else:
-		return default
-
-def fetch_uuid(key, uuid_cache):
-	return add_uuid({'uid':key}, uuid_cache=uuid_cache)['uuid']
-
-@functools.lru_cache(maxsize=128000)
-def get_or_set_uuid(uid, uuid_cache):
-	with Exclusive(uuid_cache):
-		s = 'SELECT uuid FROM mapping WHERE key = :uid'
-		res = uuid_cache.execute(s, uid=uid)
-		row = res.fetchone()
-		if row is None:
-			uu = str(uuid.uuid4())
-			c = 'INSERT OR IGNORE INTO mapping (key, uuid) VALUES (:uid, :uuid)'
-			uuid_cache.execute(c, uid, uu)
-			res = uuid_cache.execute(s, uid=uid)
-			row = res.fetchone()
-			if row is None:
-				raise Exception('Failed to add and access a new UUID for key %r' % (uid,))
-			return uu
-		else:
-			uu = row[0]
-		return uu
-
-# This is so far just internal
-@use('uuid_cache')
-def add_uuid(thing: dict, uuid_cache=None):
```

Comment:
Can you please leave this in until I can rework the AATA code?

Reply:
Ooops, sorry! Will revert

Reply:
Also needs `settings.uuid_cache_engine` restored.